presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixInitializer.java

```diff
@@ -143,6 +153,28 @@ private void waitForCoordinator()
     }
 
     private void startRubix()
+    {
+        Configuration configuration = getRubixConfiguration();
+
+        MetricRegistry metricRegistry = new MetricRegistry();
+        bookKeeperServer = new BookKeeperServer();
+        BookKeeper bookKeeper = bookKeeperServer.startServer(configuration, metricRegistry);
+        LocalDataTransferServer.startServer(configuration, metricRegistry, bookKeeper);
+
+        CachingFileSystem.setLocalBookKeeper(bookKeeper, "catalog=" + catalogName);
+        log.info("Rubix initialized successfully");
+        rubixConfigurationInitializer.initializationDone();
+    }
+
+    private void setupRubixMetrics()
+    {
+        Configuration configuration = getRubixConfiguration();
+        bookKeeperServer = new BookKeeperServer();
+        bookKeeperServer.setupServer(configuration, new MetricRegistry());
```

Comment:
The commit message is misleading. We are still startin up rubix server on coordinator to get statistics from workers.
Please be more specific.

Reply:
We don't start servers themselves. We just setup them and have metrics registered.